esp-hal/src/soc/esp32s3/cpu_control.rs

```diff
@@ -132,9 +135,11 @@ impl<'a> Drop for AppCoreGuard<'a> {
     }
 }
 
+/// Represents error that can occur in the system.
```

Comment:
```suggestion
/// Represents errors that can occur in the system.
```

To be fair, "can occur in the system" is VERY vague, especially for the single variant this enum has 🫠